CRM/Core/BAO/SchemaHandler.php

```diff
@@ -365,9 +365,9 @@ public static function changeUniqueToIndex($tableName, $dropUnique = TRUE) {
    *
    * @param $tables
    *   Tables to create index for in the format:
-   *     array('civicrm_entity_table' => 'entity_id')
```

Comment:
Note that this is just documentation but it also reflects the fact that the function expects all fields to be in an array even if just one now